docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+
+* **Rewarding participants**: the network provides rewards to incentivize those playing important roles that enable the network to function, including a) the provision of “voting rewards” to those participating in governance, and b) the provision of “node provider rewards” to those operating the node machines that are hosting the network.
+
+## Ledger
+The ICP ledger is hosted within the NNS and records all balances of ICP in the manner of a spreadsheet. Each row is called an “account,” which has two fields (i.e., there are two “columns”):
+
+* **Account identifier (bytes)**: a unique value that is derived from the identity of the “principal” that “controls” the account. Currently, the principal must either be (i) the owner of a public key pair, or (ii) a canister smart contract that is part of the NNS. Account identifiers are derived by hashing the concatenation of a domain separator, the principal ID, and the subaccount (or zeros if no subaccount is given).
+
+* **Balance** (positive integer, representing one-hundredth of a millionth of an ICP): the quantity of ICP assigned to the principal of the account.
+
+When the principal is a public key or Canister, they can apply the following operation to an account:
+
+* **Send**: send a portion of the ICP balance to another account. If all the ICP is sent to another account, then the sending account ceases to exist (i.e., is deleted from the ledger).
+* **Notify**: when the destination of the funds sent is the account of an NNS canister (e.g., an account of the governance canister), the sender can ask the ledger to notify the recipient canister of the incoming transfer. The recipient canister can then act on this notification. Two examples where this ability is used are creating a neuron and refreshing the stake of a neuron. These are detailed below.
+
+Operations that require interaction between the ledger and the governance system (Neurons):
+
+* **Create neuron**: when the principal is a public key holder, they may lock a portion of their balance inside a new neuron. Technically, creating the neuron is done in two stages. First, transfer the ICP to be staked to an account of the governance canister (which corresponds to a new neuron — the details of the association are omitted here). Then notify the governance canister of the incoming transfer which updates its internal neuron bookkeeping. If the entire balance is locked inside a new neuron, the account ceases to exist (i.e., is deleted from the ledger). To move these ICP to a different account, such as back to the original account, where they can once again be controlled like a normal balance, the associated neuron must be fully dissolved and disbursed (destroyed). The new neuron that has been created is controlled by the private key of the principal that created it.
```

Comment:
```suggestion
* **Create neuron**: when the principal is a public key holder, they may lock a portion of their balance inside a new neuron. Technically, creating the neuron is done in two stages. First, transfer the ICP to be staked to an account of the governance canister (which corresponds to a new neuron — the details of the association are omitted here). Then notify the governance canister of the incoming transfer which updates its internal neuron bookkeeping. To move these ICP to a different account, such as back to the original account, where they can once again be controlled like a normal balance, the associated neuron must be fully dissolved and disbursed (destroyed). 
```

" If the entire balance is locked inside a new neuron, the account ceases to exist (i.e., is deleted from the ledger)." is not fully correct: the tokens will still be on the ledger, they are just in an account that is owned by governance and thus cannot be touched by the user

"The new neuron that has been created is controlled by the private key of the principal that created it." is not fully true: I can claim a neuron for someone else. It just depends which principal is used to compute the subaccount. Maybe this is too technical to exampl here?